tryci

```diff
@@ -14,8 +14,9 @@
 
 DEFAULT_DOCKERFILE=.buildbot_dockerfile_default
 
-# Arguments to enable capabilities in containers.
-CAP_ARGS="--cap-add CAP_PERFMON"
+# Arguments to enable PT and rr support in docker.
+CAP_ARGS="--cap-add CAP_PERFMON --cap-add SYS_PTRACE --security-opt seccomp=unconfined"
```

Comment:
Is `--security-opt seccomp=unconfined` actually needed? I thought it was for seccomp only, which to my knowledge isn't enabled on our CI server.

Reply:
It is yeah. This is part of the build instructions getting `rr` working under docker: https://github.com/rr-debugger/rr/wiki/Docker

Reply:
If you're worried about the implications of this, we can guard it behind an `--rr` flag?

Reply:
Though ideally I'd prefer not to! The classic scenario is I get a non-deterministic CI crash, and I want to attach to it and then run my `rr` commands. If I forgot to build the container with those flags i'd have to try and tease out the crash again.